lib/mayaUsd/ufe/private/Utils.cpp

```diff
@@ -128,15 +128,21 @@ void applyCommandRestriction(const UsdPrim& prim, const std::string& commandName
         return;
     }
 
-    auto        primSpec = MayaUsdUtils::getPrimSpecAtEditTarget(prim);
-    auto        primStack = prim.GetPrimStack();
+    SdfPrimSpecHandle       primSpec = MayaUsdUtils::getPrimSpecAtEditTarget(prim);
+    SdfPrimSpecHandleVector primStack = prim.GetPrimStack();
     std::string layerDisplayName;
     std::string message { "It is defined on another layer" };
 
     // iterate over the prim stack, starting at the highest-priority layer.
-    for (const auto& spec : primStack) {
+    for (const SdfPrimSpecHandle& spec : primStack) {
         const auto& layerName = spec->GetLayer()->GetDisplayName();
 
+        // Don't block edits if there are no authored properties (attributes, relationships, etc)
```

Comment:
This is the fix: empty def and empty over should not block edit.

Reply:
Sounds like a reasonable fix, great job @pierrebai-adsk!